internals/daemon/api.go

```diff
@@ -117,3 +117,7 @@ func v1SystemInfo(c *Command, r *http.Request, _ *userState) Response {
 	}
 	return SyncResponse(result)
 }
+
+func RegisterCommand(c *Command) {
```

Comment:
Does this describe the action as accurately as possible? (I know it comes from the struct type). The type comments reads:

```
// A Command routes a request to an individual per-verb ResponseFUnc
```

Would ```AddResponseHandler``` be a better public API description?

Reply:
Technically speaking, a Command can designate multiple ResponseFunc, because there's potentially one for every HTTP request mode. And it really is a Command that we add, not just a handler. So I think it would be more confusing to veer away from the well-defined word "Command" here. We could even just strip the noun away, considering the parameter should already make it clear what we add/register.

As for the difference between Add and Register, I think daemon.Addxxxxxxx isn't really obvious. At least to me, daemon doesn't sound like a collection. Whereas Register smells of callback or handler or something like that.